lambda_functions/automate_cubedash_db_update/test_handler.py

```diff
@@ -39,3 +39,39 @@ def test_lambda_handler(self, mock_client, mock_update_cubedash_call):
         }
         handler(event, None)
         mock_update_cubedash_call.assert_called
+
+    @patch('handler.boto3.client')
+    @patch('handler._update_cubedash_db_config')
+    def test_ows_db_update(self, mock_client, mock_update_cubedash_call):
+        event = {
+            "Records": [
+                {
+                    "Sns": {
+                        "Message": "ows_20190101\n",
+                        "Timestamp": "Timestamp",
+                        "MessageId": "Message_1234"
+                    },
+                },
+            ]
+        }
+        handler(event, None)
+        mock_update_cubedash_call.assert_called
```

Comment:
These tests aren't very good. Most of the hard work is done though.

I think it would be better to `@patch` `handler.execute_command`, and check that it's being called with the correct arguments.
PyTest and/or mock can be used to temporarily set environment variables if necessary.

Reply:
Added a new test to verify `handler.exec_command`